src/Microsoft.Docs.Template/Models/TestData.cs

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using Newtonsoft.Json;
```

Comment:
do we still need to update this test model? This could be removed since we are moving to json schema.

Reply:
I will removed it in #4694  PR